webui/react/src/services/decoder.ts

```diff
@@ -286,10 +288,10 @@ const mapV1Experiment = (
 ): types.ExperimentItem => {
   return {
     archived: data.archived,
+    description: data.description,
     endTime: data.endTime as unknown as string,
     id: data.id,
     labels: data.labels || [],
-    // capture description for use in the UI.
```

Comment:
@hamidzr can you please check if the changes done in those 2 files of this MR are ok?
- webui/react/src/services/decoder.ts
- webui/react/src/types.ts